### PR TITLE
Swipet Main Headline Section

### PR DESCRIPTION
**Pull request by egiegarciano14**
## Related Links
https://www.notion.so/Main-Headline-Section-d23b9c0f0c2e419085c6a31b23e86e6b

## Description
- Main Headline Section is base on figma

## Notes
- N/A

## Screenshots
image
image
image

### CODE REVIEW

**Comment by ume45876139** [comment]
# Comments
## PC
<details>
<summary>Chrome</summary>

スクリーンショット 2022-04-26 14 35 04

</details>

- [x] Apple download button is deformation

<details>
<summary>Safari</summary>

スクリーンショット 2022-04-26 14 38 09

</details>

- [x] Google play download button is deformation at safari

## iPad

<details>
<summary>Safari</summary>

名称未設定

</details>

- [x] Google play download button is deformation at safari

## Mobile

<details>
<summary>Safari</summary>

IMG_7978

</details>

- [ ] There is extra space at right side
- [x] Google play download button is deformation at safari



# Testing data
test device: Chrome, Safari (Mac)
test time: 1h

**Comment by ume45876139** [comment]
# Working
## iPad
<details>
<summary>Screenshot</summary>

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-26 at 17 02 07

</details>

## Mobile

<details>
<summary>Screenshot</summary>

IMG_7982

</details>

# Comment

<details>
<summary>PC</summary>

スクリーンショット 2022-04-26 16 59 33

</details>

- [x] button position is not align with above text.


# Testing data
test device: Chrome, Safari (Mac)
test time: 0.5h

**Review by ume45876139 — APPROVED** [review]
# Working
## PC
<details>
<summary>Chrome</summary>

スクリーンショット 2022-04-26 17 36 58

</details>

<details>
<summary>Safari</summary>

スクリーンショット 2022-04-26 17 37 16

</details>

## iPad

<details>
<summary>Screenshot</summary>

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-26 at 17 36 53

</details>

## Mobile

<details>
<summary>Screenshot</summary>

IMG_7983

</details>

- Deformation is nothing
- Font-family is correct
- Font color is correct
- image size is proper
- Component align is proper

# Testing data
test device: Chrome, Safari (Mac)
test time: 0.5h
comment: 6
iteration: 2
total test time: 2h

File: src/pages/components/DownloadButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```
Comment (charlotteds) [inline src/pages/components/DownloadButton.js]: Inconsistency with usage of semicolons and no semicolons.
Check other files as well.

**Review by charlotteds — APPROVED** [review]

File: src/assets/tailwind.output.css [hunk 2]
```diff
@@ -3,6 +3,12 @@
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
 
+@layer base {
+  html {
+    font-family: 'Nunito', sans-serif;
+  }
+}
+
 /* Custom Components */
```
Comment (roseaugusto) [inline src/assets/tailwind.output.css]: @egiegarciano14 hello Egie, one favor here please. Please ko move sa font-face above the `@layer base`. Sorry I put it previously below sa custom component na comment. 🙇🏼

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**